nservicebus/lifecycle/endpointstartandstop.md

```diff
@@ -72,6 +72,7 @@ When the Bus is disposed, all instances of `IWantToRunWhenBusStartsAndStops` are
 
 NOTE: `Stop()` will wait for any outstanding instances of `Start()` to complete. If an instance of `IWantToRunWhenBusStartsAndStops` needs to be long running then it must start it's own background thread. Failure to do so will prevent the bus from being disposed.
 
+NOTE: `Stop()` is not supported in Version 3
```

Comment:
Rather than use an additional note, can this not be added to the previous note? I'd phrase it as:

`Stop()` method is not supported in Versions 3 and below. 

For Version language, please see: https://github.com/particular/docs.particular.net#version-language